packages/flutter/lib/src/widgets/tap_region.dart

```diff
@@ -146,6 +146,8 @@ class TapRegionSurface extends SingleChildRenderObjectWidget {
 /// When a tap outside of a registered region or region group is detected, its
 /// [TapRegion.onTapOutside] callback is called. If the tap is outside one
 /// member of a group, but inside another, no notification is made.
+/// This method will not be called when [EditableText.canTapOutsideFocus] is set
```

Comment:
Consider moving this comment to `EditableText.onTapOutside`.

Reply:
Done